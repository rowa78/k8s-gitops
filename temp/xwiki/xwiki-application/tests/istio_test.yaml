suite: test istio
templates:
  - istio.yaml
tests:
  - it: should properly set value of VirtualService
    set:
      service.externalPort: 80
      istio.enabled: true
    documentIndex: 0
    asserts:
      - isKind:
          of: VirtualService
      - isAPIVersion:
          of: networking.istio.io/v1beta1
      - matchSnapshot:
          path: spec

