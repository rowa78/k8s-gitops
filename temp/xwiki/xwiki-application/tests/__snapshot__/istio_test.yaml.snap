should properly set value of VirtualService:
  1: |
    gateways:
      - RELEASE-NAME-xwiki-gateway
    hosts:
      - '*'
    http:
      - match:
          - uri:
              prefix: /
        route:
          - destination:
              host: RELEASE-NAME-xwiki
              port:
                number: 80
            headers:
              request:
                set:
                  x-forwarded-host: '*'
